blog/docs/articles/highlighting-value-knative-c-suite.md

```diff
@@ -1,13 +1,7 @@
----
-title: "Highlighting the Value of Knative for the C-Suite"
-linkTitle: "Highlighting the Value of Knative for the C-Suite"
-date: 2021-11-04
-description: "Deploy faster and more cost-effectively without hard-to-find, specialized expertise."
-type: "blog"
----
-
 # Highlighting the Value of Knative for the C-Suite
```

Comment:
Based on https://github.com/knative/docs/pull/4442, I think this may have been a combination of @omerbensaadon and @csantanapr